.golangci.yml

```diff
@@ -3,14 +3,6 @@
 run:
   timeout: 3m
   tests: true
-  # default is true. Enables skipping of directories:
-  #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
-  skip-dirs-use-default: true
-  skip-dirs:
-    - tests
```

Comment:
why the skip flags are removed

Reply:
@R-Santev, the `Golangci-lint` warned me that these flags are not allowed anymore in the yml configuration. Therefore:

1. skip-dirs-use-default is always true by default
2. skip-dirs is used as:
```
exclude-dirs:
    - tests
```
3. the service property is also not allowed and there was no difference running it w/o this property.